### PR TITLE
windows event loop sig handler

### PR DESCRIPTION
**Pull request by jessekrubin**
except not implemented error for windows when adding sig handler to event loop

### CODE REVIEW

File: arq/worker.py [hunk 1]
```diff
@@ -576,7 +576,10 @@ async def record_health(self) -> None:
             self._last_health_check_log = log_suffix
 
     def _add_signal_handler(self, signum: Signals, handler: Callable[[Signals], None]) -> None:
-        self.loop.add_signal_handler(signum, partial(handler, signum))
+        try:
+            self.loop.add_signal_handler(signum, partial(handler, signum))
+        except NotImplementedError:
+            logger.debug('Oy vey; Windows does not support adding a signal handler to an eventloop')
```
Comment (samuelcolvin) [inline arq/worker.py:582]:
```suggestion
            logger.debug('Windows does not support adding a signal handler to an eventloop')
```

Bit non standard and confusing.

Comment (jessekrubin) [reply arq/worker.py:582]: Solid. 

PS: windows regularly drives me bananas

**Comment by samuelcolvin** [comment]
thanks so much.

**Comment by jessekrubin** [comment]
> thanks so much.

No problemo @samuelcolvin

**Pull request merged**